src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -0,0 +1,171 @@
+= Testing templates
+:page-slug: testing-templates
+:page-description: Guide to testing Koji templates.
+
+There are many powerful testing tools built into the Koji platform.
+Typically, you will begin testing in the Koji editor.
```

Comment:
This intro still could use a bit of work. It's not enough to say the tools are powerful; that has more of a marketing tone. In the dev docs, it would be more appropriate to summarize what makes them powerful for Koji developers - namely, they let you test Koji features, such as remixing...